go.cap

```diff
@@ -1,10 +1,8 @@
 github.com/onflow/cadence ()
 
 github.com/davecgh/go-spew/spew (file)
-github.com/onflow/cadence/runtime/interpreter ()
```

Comment:
is this expected ?

Reply:
I guess no entry is equivalent to `()`? @fxamacker do you maybe know?